src/app/components/JumpTo/index.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import JumpTo, { JumpToProps } from '.';
+import metadata from './metadata.json';
+import readme from './README.md';
+
+const Component = ({ jumpToHeadings = [] }: JumpToProps) => {
+  return <JumpTo jumpToHeadings={jumpToHeadings} />;
+};
+
+export default {
+  title: 'Components/JumpTo',
+  Component,
+  parameters: {
+    docs: { readme },
+    metadata,
+  },
+};
+
+// wip sample story - should use a fixture here instead probably for the titles
+export const Example = () => {
+  const jumpToHeadings = [
+    {
+      heading: 'This is a subheading - 1',
```

Comment:
Can we update this fixture data so there aren't numbers in it please? Makes it confusing to listen to on a screen reader, as some will announce a list item number. Could put a, b, c...